### ISSUE TITLE
Processing multiple assemblies

### ISSUE THREAD

**Issue by yrtimiD** [kyleherzog/AssemblyInfoReaderTask#1]
Currently, if multiple assemblies are discovered, same variables are overridden with last assembly values, which doesn't make sense to me, I'd stop on first.

Can you add an "Assembly name" prefix to this variables, so for project Common.Logging@1.2.3
variables will be:
```
Common_Logging.AssemblyInfo.AssemblyTitle = 'Common.Logging'
Common_Logging.AssemblyInfo.AssemblyVersion = '1.2.3'
```
this can be configurable to not break current users.

**Comment by kyleherzog** [comment]
I like the idea but adding the logic to pull in assembly name would add quite a bit of complexity to the project.  I believe the assembly name would need to be parsed from the XML of the project file.  Right now we are only doing a simple RegEx parse of AssemblyInfo files.

Is there a business need for making the variables available from multiple AssemblyInfo files?  For most scenarios that I've seen, when there are multiple AssemblyInfo files, only one is actually the needed.  Typically the others are test assemblies.  If the variables are only going to be utilized from one assembly, users can set the "Path to AssemblyInfo.cs" parameter to the specific path to the one desired.

**Comment by yrtimiD** [comment]
In my case I need to compose a string from versions of several assemblies in solution. Afterwards it is used for applying source control label. So I add several tasks with different paths and catch "AssemblyInfo.AssemblyVersion" value after it. The most annoying issue for me is that same variables will become overridden by any additional task instance. 
On second thoughts, maybe just adding configurable prefix for produced vars will be less noisy, more useful and will keep backward compatibility.

**Comment by kyleherzog** [comment]
Adding a prefix setting to the task should be pretty straight forward.   Adding this feature request to my planned work.

**Comment by kyleherzog** [comment]
The option to add a prefix to variables generated is now available with version 1.1
